### ISSUE TITLE
unable to build an image with EHF

### ISSUE THREAD

**Issue by chen23** [f5devcentral/f5-bigip-image-generator#4]
I'm trying to create a build for 14.1.2 w/ an EHF and the process is failing to complete.

```
./build-image -i /home/ubuntu/BIGIP-14.1.2-0.0.37.iso -c ~/confi
g.yml -p qcow2 -m ltm -b 2 -e /home/ubuntu/Hotfix-BIGIP-14.1.2.0.37.37-ENG.iso
Initializing global variable definitions
Initializing variable definitions for platform [qcow2]
Parsing remaining command line arguments
Parsing remaining configuration values from config file [/home/ubuntu/config.yml]
Configuration key [AWS_BUCKET] is not configurable for platform [qcow2]!  Skipping.
Configuration key [AWS_REGION] is not configurable for platform [qcow2]!  Skipping.
Assigning default values to eligible variables
Validating that required variables are present
Configuration was successfully initialized
Missing F5 public key!  Skipping signature verification for ISO [/home/ubuntu/BIGIP-14.1.2-0.0.37.iso].  Please provid
e ISO_SIG_PUBKEY as a path to the F5 public key to enable signature verification.
Missing F5 public key!  Skipping signature verification for ISO [/home/ubuntu/Hotfix-BIGIP-14.1.2.0.37.37-ENG.iso].  P
lease provide ISO_SIG_PUBKEY as a path to the F5 public key to enable signature verification.
Create image directory: /home/ubuntu/f5-bigip-image-generator/images
Starting the image build process
Extracting '/BIGIP1412/VERSION' from '/home/ubuntu/BIGIP-14.1.2-0.0.37.iso'
Successfully extracted '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0.0.37/qco$
2/ltm_2slot/VERSION' from '/home/ubuntu/BIGIP-14.1.2-0.0.37.iso'.
PRODUCT_NAME: BIGIP
PRODUCT_BUILD: 0.0.37
PRODUCT_VERSION: 14.1.2
Extracting '/BIGIP1412/install/ve.info.json' from '/home/ubuntu/BIGIP-14.1.2-0.0.37.iso'
Successfully extracted '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0.0.37/qco$
2/ltm_2slot/ve.info.json' from '/home/ubuntu/BIGIP-14.1.2-0.0.37.iso'.
CPU supports virtualization.
------======[ Starting disk generation for 'qcow2' 'ltm' '2' boot-locations. ]======------
Skipping checksum verification for earlier runs as json '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../$
rtifacts/BIGIP-14.1.2-0.0.37/qcow2/ltm_2slot/prepare_raw_disk.json' doesn't exist.
BIG-IP Disk Size =37
Creating 37G disk '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0.0.37/qcow2/lt$
_2slot/BIGIP-14.1.2-0.0.37.LTM.qcow2.raw' in format 'raw' -- start time: 20:28:48
Formatting '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0.0.37/qcow2/ltm_2slot$
BIGIP-14.1.2-0.0.37.LTM.qcow2.raw', fmt=raw size=39728447488
Creating 37G disk '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0.0.37/qcow2/lt$
_2slot/BIGIP-14.1.2-0.0.37.LTM.qcow2.raw' in format 'raw' -- elapsed time: 0:00:00
Wrote create_raw_disk status to '/home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.1.2-0$
0.37/qcow2/ltm_2slot/create_raw_disk.json'.
Inserting VM installation environment for 'RTM'
Skipped copying post-install in initrd as this is an RTM + HF install.
Append the new files in INITRD image: /home/ubuntu/f5-bigip-image-generator/src/lib/bash/../../../artifacts/BIGIP-14.$
.2-0.0.37/qcow2/ltm_2slot/tmp.VEkRAzZpIr/boot/initrd
3 blocks
Inserting VM installation environment for 'RTM' -- elapsed time: 0:00:34
qemu-system installing RTM Image -- start time: 20:29:22
.....................................................................................................................$
....
qemu-system installing RTM Image -- elapsed time: 0:10:10
Usage: update_initrd_image <install-mode> <ISO> <boot-initrd-img path>  <skip-post-install> <raw-disk-json>
```

log file

[image-qcow2-ltm-2slot.txt](https://github.com/f5devcentral/f5-bigip-image-generator/files/3607387/image-qcow2-ltm-2slot.txt)

**Comment by chen23** [comment]
also see PR #5.  I believe the issue is related to a missing argument for "update_initrd_image"

**Comment by alaari-f5** [comment]
fixed in rel 1.1
